docs/interpreter/cassandra.md

```diff
@@ -793,7 +812,17 @@ Below are the configuration parameters and their default value.
 
 <hr/>
 
-## 14. Bugs & Contacts
+## 14. Change Log
+
+**2.0** : 
```

Comment:
what I mean is the interpreter jar or code itself does not have 1.0 or 2.0 version though? how would a user know which one they have? or which binary release of zeppelin has which?